<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreenClass</class>
 <widget class="QMainWindow" name="SplashScreenClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SplashScreen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: #4C9FEB;
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="titleLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>661</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFF;</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;STARDEW VALLEY&lt;/strong&gt; APP</string>
       </property>
      </widget>
      <widget class="QLabel" name="loadingLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>260</y>
         <width>661</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #8FB7D6;</string>
       </property>
       <property name="text">
        <string>loading...</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>230</y>
         <width>581</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
	background-color: #2D648D;
	color: #DFEDF6;
	border-style: none;
	border-radius: 10px;
	text-align: center;
}

QProgressBar::chunk {
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(45, 100, 141, 255), stop: 1 rgba(223, 237, 246, 255));
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="creditsLabel">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>350</y>
         <width>301</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #2D648D;</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Created&lt;/strong&gt;: Razvan Calauz</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="descriptionLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>661</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #F0F6F9;</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;APP&lt;/strong&gt; DESCRIPTION</string>
       </property>
      </widget>
      <widget class="QLabel" name="versionLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>311</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #2D648D;</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Version&lt;/strong&gt; 1.0.3</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
